ocaml/database/master_connection.ml

```diff
@@ -26,6 +26,9 @@ open D
 
 let my_connection : Stunnel.t option ref = ref None
 
+let is_slave = ref (fun () -> false)
```

Comment:
A massive pain - they're used in every single db read/write operation. But agreed on the raising exceptions/logging suggestion.